lukasz-lapczyk/recruitment-web-app
# Added admin, removed sign up option

This closes #2 

# Overview
Added new type of user - admin. Admin can delete other users' accounts while not being able to delete himself/herself. Removed sign up option for creating new accounts.

# Testing prerequisites
- bundle install
- db migrations
- db seeds

# Testing
- navigate to root directory
- [ ] log in as user_1@example.org, password: Secret19
- [ ] trash icon should not appear on users list
- navigate to root directory
- [ ] log in as admin@example.org, password: Secret09
- [ ] trash icon should appear next to each user email on users list (except himself)
- [ ] account should be deleted after clicking on trash icon and confirming
